src/main/resources/templates/pdf.css

```diff
@@ -171,3 +171,18 @@ body {
     color: #eaeaea;
     transform: rotate(-45deg);
 }
+
+sub, sup {
+    font-size: 75%;
+    line-height: 0;
+    position: relative;
+    vertical-align: baseline;
+}
+
+sup {
+    top: -0.5em;
+}
+
+sub {
```

Comment:
There is no `<sub>` tag used in this PR, why do we need the CSS for it?

Reply:
Thanks for pointing to, removed.